Dockerfile

```diff
@@ -4,11 +4,6 @@ LABEL maintainer="tkukral@mirantis.com"
 # prepare directory
 WORKDIR /code
 
-# install dependencies
```

Comment:
Please don't remove this. These requirements are required fot LDAP to be working.

Reply:
oops, I merged something wrong